hazelcast/src/test/java/com/hazelcast/sql/impl/SqlTestSupport.java

```diff
@@ -64,6 +67,10 @@
  * Helper test classes.
  */
 public class SqlTestSupport extends HazelcastTestSupport {
+
+    @ClassRule
+    public static OverridePropertyRule enableJetRule = OverridePropertyRule.set("hz.jet.enabled", "true");
```

Comment:
I just couldn't find how this property is used. I would appreciate if you explain it to me before merging

Reply:
This sets the system property for the test cases. Set to `true` it will enable the Jet engine for Hazelcast instances started within the tests.

Reply:
When I wrote this comment, I couldn't understand the resolution of this property since no hz property with this name is defined in the source side. Then, I realized it came with this https://docs.hazelcast.com/imdg/4.2/configuration/overriding-configuration-settings.html feature.